### PR TITLE
test: Remove empty TEST_OUTPUT directives

### PR DESCRIPTION
**Pull request by ibuclaw**
The testsuite complains about any compiler message by default, so these are redundant.

@MoonlightSentinel  correct me if my memory is wrong here.

### CODE REVIEW

**Review by MoonlightSentinel — APPROVED** [review]
> @MoonlightSentinel correct me if my memory is wrong here.

Correct, no output is the default now

**Comment by ibuclaw** [comment]
```
gpg: Signature made Fri 01 Jan 2021 03:10:02 PM UTC using RSA key ID B9329CF8
gpg: Can't check signature: public key not found
Invalid signature http://downloads.dlang.org/releases/2.x/2.095.0/dmd.2.095.0.linux.tar.xz.sig
```
:-(

File: test/compilable/readmodify_structclass.d [hunk 1]
```diff
@@ -1,9 +1,4 @@
-/*
-REQUIRED_ARGS:
-TEST_OUTPUT:
----
----
-*/
+// REQUIRED_ARGS:
```
Comment (RazvanN7) [inline test/compilable/readmodify_structclass.d:1]: Is `REQUIRED_ARGS` necessary in this case? Can't we delete that one also?

**Review by RazvanN7 — APPROVED** [review]

**Comment by RazvanN7** [comment]
@ibuclaw The signature issue will most likely be fixed today.

File: test/compilable/readmodify_structclass.d (hunk 1 above)
Comment (ibuclaw) [reply test/compilable/readmodify_structclass.d:1]: I thought it was only fail_compilation and runnable where it was ignored, and compilable tests use the default permutations of flags unless overridden.

File: test/compilable/json.d [hunk 2]
```diff
@@ -2,10 +2,6 @@
 // REQUIRED_ARGS: -d -preview=dip1000 -o- -X -Xf${RESULTS_DIR}/compilable/json.out
 // POST_SCRIPT: compilable/extra-files/json-postscript.sh
 // EXTRA_FILES: imports/jsonimport1.d imports/jsonimport2.d imports/jsonimport3.d imports/jsonimport4.d
-/* TEST_OUTPUT:
```
Comment (RazvanN7) [inline test/compilable/json.d:5]: This test seems to be failing for some reason.

Comment (RazvanN7) [reply test/compilable/json.d:5]: @ibuclaw The issue is that removing those comments is going to affect the output of the json in terms of line numbers and therefore the expected output of this test also needs to be modifed.

Comment (RazvanN7) [reply test/compilable/json.d:5]: It's weird that circleCI passes though.

**Comment by ibuclaw** [comment]
Rebased.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**